README.md

```diff
@@ -1 +1,43 @@
-# leapp-report-analyzer
\ No newline at end of file
+# Leapp Report Analyzer
+
+Leapp Report Analyzer is a command-line tool for analyzing multiple Leapp reports and grouping systems by severity and entries.
+The tool processes JSON files containing Leapp reports and generates a JSON file and an HTML table displaying the results.
+
+By analyzing Leapp reports and providing organized output, the Leapp Report Analyzer simplifies the process of identifying and addressing potential issues in your systems.
+
+### Requirements
+
+Script works on both Python2 and Python3, tested with:
+
+- Python 2.7
+- Python 3.3
+
+### Usage
```

Comment:
you could just paste here `python3 leapp-report-analyzer.py -h` or `python leapp-report-analyzer.py -h`